kornia/geometry/transform/imgwarp.py

```diff
@@ -85,15 +79,37 @@ def warp_perspective(src: torch.Tensor, M: torch.Tensor, dsize: Tuple[int, int],
         raise ValueError("Input M must be a Bx3x3 tensor. Got {}"
                          .format(M.shape))
 
-    # launches the warper
-    h, w = src.shape[-2:]
-    return transform_warp_impl(src, M, (h, w), dsize, flags, border_mode, align_corners)
+    # TODO: remove in kornia v0.6
+    message: str = (
+        "The align_corners flag has been inverted. By default now is set True "
+        "in order to match cv2.warpAffine. Set to False in order to keep your previous behaviour. "
+        "This warning will disappear in kornia >0.6.")
+    warnings.warn(message)
```

Comment:
We should include a comment that the normal user probably does not need to take action. I imagine most people simply use the default value of `align_corners` without caring about the internals.